### ISSUE TITLE
Android SDK root and JDK path being overwritten

### ISSUE THREAD

**Issue by Soviut** [jenkinsci/unity3d-plugin#9]
From what I can tell, the plugin modifies the EditorPrefs plist file, or the EditorPrefs directly, which removes the Android SDK and JDK path on OSX. The plist file on OSX is located at `~/Library/Preferences/com.unity3d.UnityEditor.plist`

I tried creating a build step that uses PListBuddy to write the two paths into the plist file on each build, but this seemed to get stomped by the Unity3D build plugin.

The workaround I used was to set the EditorPrefs directly in my BuildScript.cs

```C#
EditorPrefs.SetString("JdkPath",        "/Library/Java/JavaVirtualMachines/jdk1.8.0_91.jdk/Contents/Home");
EditorPrefs.SetString("AndroidSdkRoot", "/Users/Shared/Jenkins/Library/Android/sdk");
```

**Comment by crystalbyte** [comment]
Has this been resolved since then?

**Comment by jjfmarket** [comment]
I don't think this is related to the plugin, because I'm seeing the same behaviour running unity via a command line script in my jenkins install
